src/main/java/org/myrobotlab/service/OakD.java

```diff
@@ -0,0 +1,42 @@
+package org.myrobotlab.service;
+
+import org.myrobotlab.framework.Service;
+import org.myrobotlab.logging.Level;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.logging.LoggingFactory;
+import org.myrobotlab.service.config.ServiceConfig;
+import org.myrobotlab.service.config.ServoConfig;
+import org.slf4j.Logger;
+/**
+ * 
+ * 
+ * 
+ * 
+ * @author GroG
+ *
+ */
+public class OakD extends Service {

```

Comment:
not a whole lot here.. maybe best to leave it out of the PR until there's an actual implementation?

Reply:
I'll look into removing it ... not super high priority though .. its a good reminder that we want an implementation for this one - since we have a group that has the hardware